ament_cmake_google_benchmark/ament_cmake_google_benchmark-extras.cmake

```diff
@@ -16,6 +16,9 @@ macro(_ament_cmake_google_benchmark_find_benchmark)
   if(NOT DEFINED _AMENT_CMAKE_GOOGLE_BENCHMARK_FIND_BENCHMARK)
     set(_AMENT_CMAKE_GOOGLE_BENCHMARK_FIND_BENCHMARK TRUE)
 
+    option(AMENT_RUN_PERFORMANCE_TESTS
+      "When not set to true, performance tests are unconditionally skipped" OFF)
```

Comment:
I would suggest to avoid multiple negations since it is hard to read and understand what value means what.

Reply:
aed665d